### DISCUSSION TITLE
Manually Insert Document using dapper SQL

### DISCUSSION THREAD

**Discussion by guintujoshua** [OrchardCMS/OrchardCore#17037] [Q&A]
I was trying to manually insert in CATDocument an item with all its  proper Index to other table such as ContentItemIndex, my problem is  that CATDocument is not using Identity so i check for the last item's Id, and add it to the table but Orchard does not recognize this so when i use Orchard's UI to add item, it fail with Duplicate PK, because its trying to add Id 4 when i already added that manually with an Stored Procedure. 

Question: Is there anyway so that i can make the table CATDocument have Identity or for me to Update Orchard last known Id so that its Sync with my manually added Items?

**Comment by Piedone** [comment]
What's CATDocument for, is this one row for each content item? Because then you can have that as an index record, and let YesSql fully manage it.

**Reply by guintujoshua** [reply]
actually i found where Orchard store its next Id, which is on the table named Identifier but what orchard do is it increment it by 20, and pre assigned the 20 next id. and i dont understand how to remove this or at least make it update every increment of1 document.

Edit: CatDocument is where orcard store the big JSON of the Content
